src/EnergyPlus/DaylightingManager.cc

```diff
@@ -6884,7 +6862,8 @@ namespace EnergyPlus::DaylightingManager {
                     ++count;
                     // need to map back to the original order of the "loop" to not change all the other data structures
                     loop = state.dataDaylightingData->ZoneDaylight(ZoneNum).MapShdOrdToLoopNum(count);
-                    if (SurfWinShadingFlag(IWin) < 10 && SurfWinShadingFlag(IWin) != SwitchableGlazing) continue;
+                    // if (SurfWinShadingFlag(IWin) <= BGBlind && SurfWinShadingFlag(IWin) != SwitchableGlazing) {
+                    if (NOT_SHADED(SurfWinShadingFlag(IWin)) || ANY_SHADE_SCREEN(SurfWinShadingFlag(IWin)) || ANY_BLIND(SurfWinShadingFlag(IWin))) continue;
```

Comment:
Is this also just checking if it's not in a condition state?

Reply:
Again, when `ifConditionallyOn` or `SwitchableGlazing`.

Reply:
(honestly I don't get the `ifConditionallyOn` or `SwitchableGlazing` logic that happens in a lot of places)